cloudify/_compat.py

```diff
@@ -33,6 +33,10 @@ def reraise(exception_type, value, traceback):
     raise exception_type, value, traceback
 """)
     text_type = unicode
+    exec("""
```

Comment:
exec-ing `def exec_` to make `exec_`, because otherwise the `exec code in globs` is a syntaxerror when imported under py3